kagiso_auth/tests/unit/test_models.py

```diff
@@ -520,3 +521,41 @@ def test_record_sign_out_invalid_status_code_raises(self):
         with pytest.raises(AuthAPIUnexpectedStatusCode):
             did_sign_out = user.record_sign_out()
             assert not did_sign_out
+
+    def test_age_returns_none_when_profile_is_none(self):
+        user = KagisoUser(
+            email='test@email.com',
+        )
+
+        assert user.age is None
+
+    def test_age_returns_none_when_birth_date_doesnt_exist(self):
+        user = KagisoUser(
+            email='test@email.com',
+        )
+
+        assert user.age is None
+
+    def test_age_returns_user_age_when_birthday_passed(self):
+        user = KagisoUser(
```

Comment:
The names of these two last tests are not ideal.  Just from the names it seems that there is a function called 'age' that returns the age of a user. However comparing the names of these two tests, I get the impression that 'age' should either return or not return a user's age based on whether his birthday is upcoming (or passed).